clients/clearly-defined/src/main/kotlin/ClearlyDefinedService.kt

```diff
@@ -29,7 +29,6 @@ import kotlinx.coroutines.awaitAll
 import kotlinx.coroutines.runBlocking
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
-import kotlinx.serialization.decodeFromString
```

Comment:
Hmm, interesting that these were not found by neither detekt nor Qodana. Any idea why that is, @fviernau?

Reply:
I don't know it for sure. But this file uses a function named `decodeFromString()` but from a different package.